recipes/nni/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "nni" %}
+{% set version = "2.10" %}
+{% set sha256 = "826702c4165731c044d30af49230a761ffc753bda8f6be38be9be642fd367037" %}
```

Comment:
There's a problem with this value apparently!? https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=31595&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=4dddc55b-10bc-50fc-ac68-7ff899b7a030&l=461